server/scripts/websocket.js

```diff
@@ -25,7 +25,7 @@ wss.on('connection', function(ws, req) {
   db.laps()
   .then((laps) => {
     // console.log(laps)
-    for (let lap = 0; lap < laps.length; lap++) {
+    for (let lap = laps.length - 1; lap >= 0; --lap) {
```

Comment:
Test to make sure all data rows are being displayed

Reply:
Yup! It works. In the context of a for loop, they act the same.